### PR TITLE
kubelet/rkt: Add RunPod() for rkt.

### PR DESCRIPTION
**Pull request by yifan-gu**
@vmarmol @dchen1107 @jonboulle @bakins

This has a dependency on some previous PR, (makePodServiceFileName).

Thanks you!

### CODE REVIEW

File: pkg/kubelet/rkt/rkt.go [hunk 1]
```diff
@@ -138,3 +146,146 @@ func (r *Runtime) runCommand(args ...string) ([]string, error) {
...
+
+	runPrepared := fmt.Sprintf("%s run-prepared --private-net=%v %s", r.absPath, pod.Spec.HostNetwork, uuid)
+	units := []*unit.UnitOption{
+		newUnitOption(unitKubernetesSection, unitRktID, uuid),
+		newUnitOption(unitKubernetesSection, unitPodName, string(b)),
+		newUnitOption("Service", "ExecStart", runPrepared),
+	}
+
+	// Save the unit file under systemd's service directory.
+	// TODO(yifan) Garbage collect 'dead' serivce files.
```
Comment (vmarmol) [inline pkg/kubelet/rkt/rkt.go]: typo: service

**Comment by vmarmol** [comment]
Looks good, we should wait for the deps to go in first though :)

**Comment by yifan-gu** [comment]
@vmarmol Rebased. Think still will conflict with others (for the import path)

**Comment by vmarmol** [comment]
LGTM, it may be okay alongside #7553

**Comment by yifan-gu** [comment]
I feel it will conflict on the kubecontainer import  :(
Will do rebase then.

**Pull request merged**